release-engineering/role-handbooks/branch-manager.md

```diff
@@ -206,47 +208,77 @@ There are more examples of the release process under the [References](#reference
 
 It is also highly recommended to glance over the [Release Commands Cheat Sheet](https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/patch-release-team.md#release-commands-cheat-sheet).
 
-**General bookkeeping**:
+### Creating and managing the Release Issue
 
 Prior to cutting a release version, [open a "Cut a Release" issue](https://github.com/kubernetes/sig-release/issues/new?template=cut-release.md&title=Cut+1.x.y-%7Balpha%2Cbeta%2Crc%7D.z+release) on [kubernetes/sig-release](https://github.com/kubernetes/sig-release).
 
-On the issue template, there are comments describing the predefined items that need to be completed.
+On the issue template, there are comments describing the predefined items that 
+need to be completed.
+
+#### Testgrid Screenshots
 
 For the item `Screenshot unhealthy release branch testgrid boards...`:
 
-You can use the `testgridshot` tool to create screenshots of unhealthy testgrid boards. `testgridshot` also takes care of uploading the screenshots to the `k8s-staging-releng` bucket, and generating the markdown table that you can post as a comment to the issue.
+`krel testgridshot` takes care of generating screenshots of 
+[Testgrid](https://testgrid.k8s.io/) to keep as a reference of the state 
+it was in before cutting a release. This subcommand will generate images 
+of the boards and upload them to a Cloud Storage bucket, as well as the
+Markdown code.
 
-`testgridshot` is a Bash script located in the [kubernetes/release] repository. By default, it takes a screenshots of **blocking** and **informing** testgrid boards that have jobs that are **failing**. If you want to customize this, open the script in your preferred text editor and see the comment at the beginning of the script for possible customizations.
+To invoke the subcommand, run it with the branch you are working on:
 
-The script can be invoked in the following way:
+```
+krel testgridshot --branch 1.20
+```
+
+You can include other testgrid states in the output and even have krel
+autocomment the issue for you:
 
-```shell
-./testgridshot <release>
 ```
+krel testgridshot --branch 1.20 --github-issue 12345 --states=FLAKY
+```
+
+Once the script generates the Markdown table, post it as a comment on the created issue. You can take a look at the [following comment](https://github.com/kubernetes/sig-release/issues/1249#issuecomment-696702503) as an example.
+
+#### Adding data about the Cloud Build Jobs
+
+When running a release cut, you should open a thread in the [#release-management] 
```

Comment:
`[#release-management]` is not linked to anything

Reply:
Done ;:)